src/main/java/GeneralService/Configurations.java

```diff
@@ -40,4 +41,23 @@ public List<OnlineService> activatedServices() {
         }
         return activatedServices;
     }
+
+    public List<String> getValidCarUserIds() {
```

Comment:
this method is not needed, it's only used in one place, move it to validCarUserIds